packages/notifications/push/amplify_push_notifications/lib/src/amplify_push_notifications_impl.dart

```diff
@@ -296,15 +306,15 @@ abstract class AmplifyPushNotifications
   }
 
   @override
-  Future<void> setBadgeCount(int badgeCount) async {
+  void setBadgeCount(int badgeCount) {
```

Comment:
What's the reason for this change? If `_hostApi.setBadgeCount` throws it will be uncaught. You can change it to: `_hostApi.setBadgetCount(badgeCount).ignore()` to specifically ignore any exceptions (if we're sure this will always succeed) or change this back to Future

Reply:
I think the reason behind this is that on iOS the `applicationBadgeIconNumber` setter is sync, and it doesn't throw, so on iOS it always succeeds; RN equivalent API is sync. Overall developers don't need to await for this action.

`_hostApi.setBadgeCount` impl on Android is guarded not to be called here, so we are not expecting it to throw any exception. But it may be a good idea to use `.ignore()` to ensure "developers don't need to await for this action".

Reply:
Added` ignore()`